rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -29,6 +30,7 @@ public class FederationStorageProviderImpl implements FederationStorageProvider
     private Federation oldFederation;
     private boolean shouldSaveOldFederation = false;
 
+    private Federation proposedFederation;
```

Comment:
Should we add these 2 fields at this time? They are not used

Reply:
Done